contracts/Registry.sol

```diff
@@ -39,6 +39,8 @@ contract Registry is IRegistry, Governable {
     address internal _farmController;
     // Locker contract.
     address internal _locker;
+    // CoverageDataProvider contract
+    address internal _coverageDataProvider;
```

Comment:
in this version of Registry the list of supported contracts is set on construction. we should change it to support whatever future contracts we may have, not necessarily in this PR

Reply:
imo, this would be a good task for the new team members.